src/components/SearchShopByTag.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <div class="search-by-tag">
+        <select v-on:change="showSearchResults($event)">
+            <option v-for="tag in tags" :value='tag'>{{ tag }}</option>
+        </select>
+        <shop-list :shops="shops" />
+   </div>
+</template>
+
+<script>
+import { requestSearchByTag } from '@/io/io';
+import ShopList from '@/components/ShopList';
+
+export default
+{
+    name: 'search-shop-by-tag',
+    data ()
+    {
+        return {
+            query: '',
+            shops: [],
+            // TODO:
+            // tagsArray:
+            // hard coded for basic implementation
+            // set to be fetched from server
+            tags: [
+                'florist',
+                'grocery',
+                'pharmacy',
+                'newsstand',
+                'butcher',
+                'bakery',
+                'cafe',
+                'coffeshop',
+                'clothes',
+                'furnishings',
+            ],
+        };
+    },
+    methods:
+    {
+        showSearchResults (event)
+        {
+            requestSearchByTag(event.target.value)
```

Comment:
This is considered hacky. Event should be accessed only if event is concerned.
See [Vue documentation](https://vuejs.org/v2/guide/forms.html#Select) for further reference.

Reply:
agree.